src/ui/utils/prefs.ts

```diff
@@ -85,4 +85,5 @@ export const asyncStore = asyncStoreHelper("devtools", {
   eventListenerBreakpoints: ["event-listener-breakpoints", undefined],
   replaySessions: ["replay-sessions", {}],
   commandHistory: ["command-history", []],
+  tabs: ["tabs", []],
```

Comment:
where is this used if there is a replay session state?

Reply:
nowhere 👍